.vscode/settings.json

```diff
@@ -30,6 +30,6 @@
   "prettier.endOfLine": "lf",
   "workbench.colorCustomizations": {},
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
Please revert vscode changes

Reply:
I can do that if you want. 

But just as a heads up: this is an automatic change that VSCode is making in all projects right now. This variable is no longer boolean, and now uses the string value instead.

Every time you open your VSCode it will automatically change this property again.

See: https://stackoverflow.com/a/77637765